pkg/ddc/alluxio/types.go

```diff
@@ -20,7 +20,7 @@ import (
 	"time"
 
 	"github.com/fluid-cloudnative/fluid/pkg/common"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I think there is no need to change this.

Reply:
vscode will auto add, i will fix it ：)